src/mongo/MongoScrapbook.ts

```diff
@@ -0,0 +1,175 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { ParseTree } from 'antlr4ts/tree/ParseTree';
+import { ANTLRInputStream as InputStream } from 'antlr4ts/ANTLRInputStream';
+import { CommonTokenStream } from 'antlr4ts/CommonTokenStream';
+import * as mongoParser from './grammar/mongoParser';
+import { MongoVisitor } from './grammar/visitors';
+import { mongoLexer } from './grammar/mongoLexer';
+import * as vscodeUtil from './../utils/vscodeUtils';
+import { CosmosEditorManager } from '../CosmosEditorManager';
+import { IAzureParentNode, AzureTreeDataProvider, IActionContext } from 'vscode-azureextensionui';
+import { MongoFindResultEditor } from './editors/MongoFindResultEditor';
+import { MongoFindOneResultEditor } from './editors/MongoFindOneResultEditor';
+import { MongoCommand } from './MongoCommand';
+import { MongoDatabaseTreeItem } from './tree/MongoDatabaseTreeItem';
+
+const output = vscodeUtil.getOutputChannel();
+const notInScrapbookMessage = "You must have a MongoDB scrapbook (*.mongo) open to run a MongoDB command.";
+
+export async function executeAllCommandsFromActiveEditor(database: IAzureParentNode<MongoDatabaseTreeItem>, extensionPath, editorManager: CosmosEditorManager, tree: AzureTreeDataProvider, context: IActionContext): Promise<void> {
+	output.appendLine("Running all commands in scrapbook...")
```

Comment:
Separate PR

Reply:
Looked at those with tabs, don't think this is worth messing with. The default VSCode settings at least keep from overwriting willy nilly.

Reply:
I don't expect you to go back and change old files from tabs to spaces. However, I do prefer that new files follow our established pattern/preference, which is spaces. Since you already merged this PR it's no longer a "new file" and I agree it's not worth a separate PR.